crates/sqllib/src/string.rs

```diff
@@ -331,7 +403,7 @@ pub fn repeat__(value: SqlString, count: i32) -> SqlString {
     if count <= 0 {
         SqlString::default()
     } else {
-        SqlString::from(value.str().repeat(count as usize))
+        SqlString(ArcStr::repeat(value.str(), count as usize))
```

Comment:
how do you even discover there functions?

Reply:
I studied the ArcStr API because it is frustratingly bare.